setup.py

```diff
@@ -13,26 +13,8 @@
 else:
     use_scm_version = True
 
-with open("README.md") as f:
-    long_description = f.read()
-
 setup(
-    name="poglink",
     version=version,
     use_scm_version=use_scm_version,
     setup_requires=["setuptools_scm"],
```

Comment:
Pretty barebones now, just needs to be here for legacy support and for this dynamic stuff.